osf/models/mixins.py

```diff
@@ -1025,7 +1037,9 @@ class Meta:
     reviews_comments_private = models.BooleanField(null=True, blank=True)
     reviews_comments_anonymous = models.BooleanField(null=True, blank=True)
 
-    DEFAULT_SUBSCRIPTIONS = ['new_pending_submissions']
+    DEFAULT_SUBSCRIPTIONS = [
```

Comment:
@Johnetordoff Maybe just get rid of this member, seems to interrupt bussness logic.